Tests/Test-IsSubdirectoryOf.Tests.ps1

```diff
@@ -0,0 +1,51 @@
+Import-Module -Force $PSScriptRoot/../Docker.Build.psm1
+. "$PSScriptRoot\..\Private\Test-IsSubdirectoryOf.ps1"
+. "$PSScriptRoot\..\Private\Format-AsAbsolutePath.ps1"
+
+Describe 'test if child path is a subdirectory of path' {
+
+    BeforeAll {
+        $parentPath = Join-Path (Split-Path -Parent $PSScriptRoot) "Test-Data"
+    }
+
+    Context 'both paths are valid directories' {
+
+        It 'succeeds when child path is part of path' {
+            $childPath = Join-Path $parentPath "ExampleRepos"
+            $result = Test-IsSubdirectoryOf -Path $parentPath -ChildPath $childPath
+            $result | Should -Be $true
+        }
+
+        It 'succeeds when deeper child path is part of path' {
+            $childPathLevel1 = Join-Path $parentPath "ExampleRepos"
+            $childPathLevel3 = Join-Path $childPathLevel1 "/3.0/servercore/amd64/"
+            $result = Test-IsSubdirectoryOf -Path $parentPath -ChildPath $childPathLevel3
+            $result | Should -Be $true
+        }
+
+        It 'fails when child path is NOT part of path' {
+            if ($IsWindows) {
+                $childPath = $Env:windir
+            } else {
+                $childPath = '/usr/bin'
+            }
```

Comment:
You could use the root directory of where we are 'located' which should never be a child of any directory. This works for all OS variants.

```suggestion
         	$childPath = Split-Path -Qualifier (Get-Location)
```

Reply:
Thanks! Works perfectly. 🌞